Eastern-Research-Group/mywaterway
# Feature/810 update dependencies

## Related Issues:
* [HMW-803](https://jira.epa.gov/browse/HMW-803)
* [HMW-810](https://jira.epa.gov/browse/HMW-810)

## Main Changes:
* Updated 3rd party dependencies.
  * Left ArcGIS JS API rolled back to 4.29. This fixes an issue with printing the page when the map is small. I looked into doing the min height, but I think the experience would be bad for users with small screens since they won't be able to see the entire map. 
* Fixed extra padding on the layer list of the state tribal map. So, this one was actually another issue with the ArcGIS JS API v4.30. I went ahead and fixed that issue for v4.30 (in `mapStyles.css` changed `.esri-view` to `.esri-view-root`). 
  * v4.29 had an issue where the layer list was full height with no scrollbar, so I fixed that issue as well. In in `mapStyles.css` changed `.esri-expand__content-container` to `.esri-expand__content`.
* Fixed issue with state tribe page not having enough padding for mobile users to easily scroll the page. 
* Fixed a cypress test number of past water conditions went up in that huc.
* Fixed an issue with the proxy not working when running locally, this broke Cyan when running locally.  

## Steps To Test:
1. Navigate to http://localhost:3000/waterbody-report/MDE_EASP/MD-02140205-Mainstem/2022
2. Use the browsers print functionality
3. Verify the app doesn't crash when print preview is shown
4. Navigate to http://localhost:3000/tribe/DELAWARENATION
5. Open the layer list
6. Verify it doesn't have any extra padding to the right and that it is scrollable
7. Switch to a mobile layout
8. Verify there is enough padding on both sides to scroll the page (should match padding on community page)
9. Test other features to verify updating dependencies didn't break anything
